fcm/README.md

```diff
@@ -0,0 +1,69 @@
+# Parse SDK Android FCM
+FCM support for Parse Android apps
+
+## Setup
+
+### Installation
+
+Add dependency to the application level `build.gradle` file.
+
+```groovy
+dependencies {
+    implementation 'com.parse:parse-android:latest.version.here'
+    implementation 'com.parse:parse-android-fcm:latest.version.here'
+
+    implementation 'com.google.firebase:firebase-messaging:12.0.1'
+}
+```
+Then, follow Google's docs for [setting up an Firebase app](https://firebase.google.com/docs/cloud-messaging/android/client). Although the steps are different for setting up FCM with Parse, it is also a good idea to read over the [Firebase FCM Setup](https://firebase.google.com/docs/cloud-messaging/android/client).
+
+You will then need to register some things in your manifest, specifically:
+```xml
+<service
```

Comment:
The parse-android-fcm should include this service in its AndroidManifest.xml file.  There should be manifest merging when building it as an AAR.